spot_rl_experiments/spot_rl/utils/img_publishers.py

```diff
@@ -622,7 +622,7 @@ def _publish(self):
                     print(f"Affordance Point is NaN for {class_label}, skipping")
                     continue
             except Exception as e:
-                print(f"Issue of predicting location: {e}")
+                print(f"Issue of predicting location for {class_label} : {e}")
```

Comment:
@jimmytyyang maybe we can disable SpotOpenVocObjectDetectorPublisher fully.

This is only being used by Nexus, and it doesn't matter to us in vizusalization there about how many objects are being detected and sent for target object highlight.

Lmk what you think

Reply:
this is used for target object highlight -- let's chat